src/charm.py

```diff
@@ -133,6 +134,18 @@ def _on_update_status(self, _):
             )
             return
 
+        if self.cluster.get_tls_statuses():
+            self.status.set_and_share_status(self.cluster.get_tls_statuses())
```

Comment:
Use ternary operator here:
```suggestion
        if (tls_statuses := self.cluster.get_tls_statuses()):
            self.status.set_and_share_status(tls_statuses)
```